windows/deploy_windows.ps1

```diff
@@ -221,9 +221,19 @@ Function Build-App
         [string] $BuildArch
     )
 
-    Invoke-Native-Command -Command "$Env:QtQmakePath" `
+	if($Env:jamulus_build_config){
```

Comment:
Even though clang-format hasn't caught you, could you please follow the style used elsewhere in the file.